lib/internal/modules/cjs/helpers.js

```diff
@@ -72,6 +72,29 @@ function stripShebang(content) {
   return content;
 }
 
+// Strip either the shebang or UTF-8 BOM of a file.
```

Comment:
"UTF-8 BOM" doesn't really make sense. The BOM is a unicode code point, not a sequence of bytes in a particular encoding of unicode. This strips the BOM just as well if the input was UTF-16 encoded.